resources/views/components/article_thumbnail_index.blade.php

```diff
@@ -0,0 +1,42 @@
+@php
+
+$url = $url ?? $article->getUrl();
+$showEduTags = $showEduTags ?? false;
+
+@endphp
+<div class="tailwind-rules">
+    <div
+        class="tw-container tw-mx-auto tw-flex tw-h-[475px] tw-max-w-screen-xl tw-flex-wrap tw-px-6 tw-py-4 md:tw-h-[400px] md:tw-flex-nowrap md:tw-py-8">
+        <div class="tw-h-1/3 tw-w-full md:tw-mr-4 md:tw-h-full md:tw-w-1/2">
+            @if ($article->main_image)
+                <a href="{{ $url }}" class="tw-h-full">
+                    <img src="{!! $article->getThumbnailImage() !!}"
```

Comment:
As long as `getThumbnailImage` is returning a non-html string, we don't have to use the `{!! !!}` brackets
```suggestion
                    <img src="{{ $article->getThumbnailImage() }}"
```